### DISCUSSION TITLE
missing definition of "getOrCreateInstance" at @types/bootstrap

### DISCUSSION THREAD

**Discussion by techfox7** [DefinitelyTyped/DefinitelyTyped#54828] [Issues with a @types package]
Hello, I found "getOrCreateInstance" definition is missing at @types/bootstrap/Alert, Carousel, Toast, ...
"getOrCreateInstance" always comes along with "getInstance" method.

Shall I change the type definition files ?

**Comment by nicolas377** [comment]
You can always make a pull request! Instructions can be found [here](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request)

**Reply by techfox7** [reply]
Yes, I am happy to contribute anyway.

**Comment by darthf1** [comment]
Resolved in `5.1.2`.
